week-2/median/median.go

```diff
@@ -0,0 +1,6 @@
+package median
+
+func GetMedian(i []int) float64 {
+	var length = len(i)
+	return float64(length) / 2 + 0.5
```

Comment:
This is not a correct median function for all the cases. Please refer https://en.wikipedia.org/wiki/Median